Assets/Project/Scripts/GamePlayScene/BoardManager.cs

```diff
@@ -95,6 +95,22 @@ public GameObject GetBottle(int tileNum)
             return _squares[x, y].bottle != null ? _squares[x, y].bottle.gameObject : null;
         }
 
+        /// <summary>
+        /// ボトルIDからボトルの現在位置を取得
+        /// </summary>
+        /// <param name="bottleId"></param>
+        public Vector2 GetBottlePosById(int bottleId)
+        {
+            foreach (var item in _squares) {
+                if (item.bottle != null && item.bottle.Id == bottleId) {
+                    return item.worldPosition;
+                }
+            }
+
+            Debug.LogError($"Cannot find bottle of ID[{bottleId}]");
+            return Vector2.zero;
```

Comment:
C# ではどういうポリシーが普通かはわかりませんが，僕は `Vector2? ` を返す関数にして，ここでは
```suggestion
            return null;
```
としますね

Reply:
C#のポリシーはわかりませんが、ここはただ  `null` チェックするのはだるかったからそうしただけなので修正します